test/live_view_native/swiftui/rules_parser_test.exs

```diff
@@ -366,6 +366,18 @@ defmodule LiveViewNative.SwiftUI.RulesParserTest do
 
       assert parse(input) == output
     end
+
+    test "gesture" do
+      input = ~s{offset(x: gesture_state(:drag, .translation.width))}
+      output = {:offset, [], [x: {:__gesture_state__, [], [:drag, {:., [], [nil, {:., [], [:translation, :width]}]}]}]}
```

Comment:
@bcardarella, can the output be the following (changes marked with `^`s)?

```elixir
output = {:offset, [], [[x: {:__gesture_state__, [], [:drag, {:., [], [nil, {:., [], [:translation, :width]}]}]}]]}
                        ^                                                                                        ^
```

The pattern so far has been to parse a key value list into a list:
```elixir
input = ~s{searchable(change: event("search-event", throttle: 10_000))}
output = {:searchable, [], [[change: {:__event__, [], ["search-event", [throttle: 10_000]]}]]}
      
input = "background(content: :star_red)"
output = {:background, [], [[content: :star_red]]}
```

Reply:
Yeah, I would prefer for the client if it followed the same format as a regular modifier would, just with `gesture_state` replaced with `__gesture_state__`

Reply:
Got it, I've pushed a change with the updated output